src/schema/rules/checks/events.yaml

```diff
@@ -43,3 +43,29 @@ SortedOnsets:
   checks:
     # n/a values will likely cause false alarms if encountered. Consider alternatives.
     - allequal(sorted(columns.onset, "numeric"), columns.onset)
+
+ImplausibleNegativeEventOnset:
+  issue:
+    code: IMPLAUSIBLE_NEGATIVE_EVENT_ONSET
+    message: |
+      Event onsets were found more than 60s before the start of recording, indicated by negative values.
+      If this is not a mistake, remove or consider an alternative method of representing these events.
+    level: warning
+  selectors:
+    - suffix == "events"
+    - extension == ".tsv"
+  checks:
+    - min(columns.onset) >= -60
+
+ImplausiblePositiveEventOnset:
+  issue:
+    code: IMPLAUSIBLE_POSITIVE_EVENT_ONSET
+    message: |
+      Event onsets were found more than a month after the start of recording.
+      If this is not a mistake, remove or consider an alternative method of representing these events.
+    level: warning
+  selectors:
+    - suffix == "events"
+    - extension == ".tsv"
+  checks:
+    - max(columns.onset) < 2678400
```

Comment:
ok, got "inspired" and added one for implausible positive one.  I think not even .1% of users should stick more than a month worth of events into one file. WDYT?

Reply:
I'm not sure these checks are necessary.  On the negative onsets, I think I have seen driving simulation experiments where there were simulator events and driver response events that were recorded in the events file before the EEG recording was turned on after the person was driving a while.  The behavioral events were recorded through the whole thing, but the onset was zeroed at the point the EEG started.

On the upper end...   I am not sure why this would be necessary either.   Consider a wearable EKG recording --- it is possible to have long term monitoring....